### PR TITLE
taxonomy: Add/update translations

### PR DESCRIPTION
**Pull request by ArturLange**
### What
Added new translations, fixed and updated old ones

### Part of
- #5690

### CODE REVIEW

File: lib/ProductOpener/Ingredients.pm [hunk 1]
```diff
@@ -200,7 +200,7 @@ my %may_contain_regexps = (
 		"Dit product kan sporen van|bevat mogelijk sporen van|Kan sporen bevatten van|Kan sporen van|bevat mogelijk|sporen van",
 	nb =>
 		"kan inneholde spor av|kan forekomme spor av|kan inneholde spor|kan forekomme spor|kan inneholde|kan forekomme",
-	pl => "może zawierać śladowe ilości|produkt może zawierać|może zawierać|możliwa obecność",
+	pl => "może zawierać śladowe ilości|produkt może zawierać:?|może zawierać:?|możliwa obecność",
```
Comment (stephanegigandet) [inline lib/ProductOpener/Ingredients.pm:203]: why did you add :? shouldn't it be just ? (if you want to make that word optional)

Comment (ArturLange) [reply lib/ProductOpener/Ingredients.pm:203]: I wanted to make ':' optional

**Comment by ArturLange** [comment]
@stephanegigandet Should be ready now :slightly_smiling_face:

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Comment by alexgarel** [comment]
I will update test result.

**Comment by alexgarel** [comment]
:clap: @ArturLange, thank you so much. 

I will add you in the team so you can start your PR directly from this project (no need of a fork)
